liquibase-standard/src/main/java/liquibase/statement/core/SelectFromDatabaseChangeLogStatement.java

```diff
@@ -71,8 +81,23 @@ public ByTag(String tagName) {
             this.tagName = tagName;
         }
 
-        public String getTagName() {
-            return tagName;
+        @Override
+        public String generateSql(Database database) {
+            return " WHERE "+database.escapeColumnName(null, null, null, "TAG")+"='" + getTagName() + "'";
```

Comment:
Could you replace those string concatenations by String.format ? in this file and in 
 SelectFromDatabaseChangeLogStatement.java  too!